src/Agent.Worker/TaskCommandExtension.cs

```diff
@@ -232,6 +236,33 @@ private void ProcessTaskUploadSummaryCommand(IExecutionContext context, string d
             }
         }
 
+        private void ProcessTaskUploadFileCommand(IExecutionContext context, string data)
+        {
+            if (!String.IsNullOrEmpty(data))
+            {
+                var filePath = Path.GetFullPath(data);
+
+                if (File.Exists(filePath))
```

Comment:
I would prefer the pattern ProcessTaskUploadSummaryCommand() had, don't do any error checking since ProcessTaskAddAttachmentCommand() will check them anyway.